xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -58,7 +62,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Declares whether this reader allows multiple root-level CBOR data items.</summary>
         <value>To be added.</value>
```

Comment:
Missing text.

Reply:
```suggestion
        <value><see langword="true"/> if this reader allows multiple root-level CBOR data items; <see langword="false"/> otherwise.</value>
```